src/linter.ts

```diff
@@ -233,8 +233,9 @@ export class Linter {
         }
     }
 
-    private getEnabledRules(configuration: IConfigurationFile = DEFAULT_CONFIG, isJs: boolean): IRule[] {
-        const ruleOptionsList = convertRuleOptions(isJs ? configuration.jsRules : configuration.rules);
+    private getEnabledRules(configuration: IConfigurationFile = parseConfigFile(DEFAULT_CONFIG), fileName: string): IRule[] {
+        const isJs = /\.jsx?$/i.test(fileName);
```

Comment:
possible to reuse `IS_JS` from rulesForFile.ts?

Reply:
Now reused.